src/com/videojs/providers/IProvider.as

```diff
@@ -26,6 +26,16 @@ package com.videojs.providers{
         function get duration():Number;
 
         /**
+         * Should return a value that indicates the current seekable start, in seconds.
+         */
+        function get seekStart():Number;
```

Comment:
I always appreciate how interface classes make folks add changes to 3 extra files :)